src/main/java/com/amazon/opendistroforelasticsearch/security/support/ConfigConstants.java

```diff
@@ -99,6 +99,8 @@
     public static final String OPENDISTRO_SECURITY_USER = OPENDISTRO_SECURITY_CONFIG_PREFIX+"user";
     public static final String OPENDISTRO_SECURITY_USER_HEADER = OPENDISTRO_SECURITY_CONFIG_PREFIX+"user_header";
 
+    public static final String OPENDISTRO_SECURITY_USER_ROLES_STRING = OPENDISTRO_SECURITY_CONFIG_PREFIX + "user_roles_string";
```

Comment:
For consistency with `injected_user` (that is also a String) please rename to remove `STRING`: 'OPENDISTRO_SECURITY_USER_ROLES_STRING` ->`OPENDISTRO_SECURITY_USER_AND_ROLES`, `"user_roles_string"` -> "user&roles" or "user_and_roles".

Reply:
Renamed. thanks.